communication/moderators.md

```diff
@@ -119,48 +130,59 @@ These are listed as "Communications Managers" in YouTube
+| Chris Aniszczyk     | @caniszczyk         | [CT - Central Time (US Central)](https://time.is/CT)    | − 06:00    |
 
 #### Moderators
 
 Moderators seats: 10
 
-- Bob Killen (@mrbobbytables) - ET
-- Ihor Dvoretskyi (@idvoretskyi) - EET
-- Jaice Singer DuMars (@jdumars) - PT
-- Jeffrey Sica (@jeefy) - ET
-- Jorge Castro (@castrojo) - ET
-- Katharine Berry (@Katharine) - PT
-- Paris Pittman (@parispittman) - PT
-- Jorge Alarcon (@alejandrox1) - ET
-- Yang Li (@idealhack) - CST
-- Noah Kantrowitz (@coderanger) - PT
+| Name                | Kubernetes Slack ID | Timezone                                                | UTC offset |
+|---------------------|---------------------|---------------------------------------------------------|------------|
+| Jorge Castro        | @castrojo           | [ET - Eastern Time (US East Coast)](https://time.is/ET) | − 05:00    |
+| Jeffrey Sica        | @jeefy              | [ET - Eastern Time (US East Coast)](https://time.is/ET) | − 05:00    |
+| Bob Killen          | @mrbobbytables      | [ET - Eastern Time (US East Coast)](https://time.is/ET) | − 05:00    |
+| Jorge Alarcon       | @alejandroxv1       | [ET - Pacific Time (US East Coast)](https://time.is/ET) | − 05:00    |
+| Katharine Berry     | @Katharine          | [PT - Pacific Time (US West Coast)](https://time.is/PT) | − 05:00    |
+| Jaice Singer DuMars | @jdumars            | [PT - Pacific Time (US West Coast)](https://time.is/PT) | − 08:00    |
+| Paris Pittman       | @paris              | [PT - Pacific Time (US West Coast)](https://time.is/PT) | − 08:00    |
+| Noah Kantrowitz     | @coderanger         | [PT - Pacific Time (US West Coast)](https://time.is/PT) | − 08:00    |
+| Ihor Dvoretskyi     | @ihor.dvoretskyi    | [EET - Easter European Time](https://time.is/EET)       | + 02:00    |
+| Yang Li             | @idealhack          | [CST - China Standard Time](https://time.is/China)      | + 08:00    |
+
 
 #### Moderators Pro Tempore
 
 Moderators pro tempore seats: 9
 
-- Dylan Graham (@DylanGraham) - AEST
-- Angus Lees (@anguslees) - AEST
-- Roy Lenferink (@rlenferink) - CET
-- Loic Le Dru (@lledru) - CET
-- Puja Abbassi (@puja108) - CET
-- Manjunath Kumatagi (@mkumatag) - IST
-- Pandiyaraja Ramamoorthy (@pontiyaraja) - IST
-- James Munnelly (@munnerz) - GMT
-- Joel Speed (@JoelSpeed) - GMT
-- Aditya Konarde (@aditya-konarde) - IST
+| Name                    | Kubernetes Slack ID | Timezone                                                        | UTC offset |
+|-------------------------|---------------------|-----------------------------------------------------------------|------------|
+| Dylan Graham            | @DylanGraham        | [AEST - Australian Eastern Standard Time](https://time.is/AEST) | − 05:00    |
+| Angus Lees              | @anguslees          | [AEST - Australian Eastern Standard Time](https://time.is/AEST) | − 05:00    |
```

Comment:
AEST is not UTC-05.  I suspect you've found (US) EST instead (a common automated error).  AEST is UTC+10

Also: the time.is links are slightly wrong.
time.is/AEST reports that we don't have daylight savings, since (technically correctly) AEST is only the UTC+10 half of the year, and it's "AEDT" for the UTC+11 half.  Compare for example [time.is/EST](https://time.is/EST) and [time.is/ET](https://time.is/ET) for another version of the same distinction.

The "correct" timezone (indicates correct daylight saving info) for me on time.is appears to be "time.is/Sydney", because oddly they don't seem to use IANA zone identifiers (although they report them).

I think Dylan is in time.is/Melbourne, although the two timezones are equivalent for all non-historical purposes.

Time is hard.

Reply:
Times are super frustrating (shakes fist at whoever came up with DST). >_< It may just be best to remove the UTC offset as that does change depending on the time of year. What about linking to "AET" sort of the equivilant for PT and ET - https://www.worldtimeserver.com/time-zones/aet/ ? They "follow" DST

Reply:
> AEST is not UTC-05. 

Copy&paste mistake, sorry =/

Reply:
> What about linking to "AET" sort of the equivilant for PT and ET - https://www.worldtimeserver.com/time-zones/aet/ ?

Works for me, and means we don't feel compelled to call out Sydney vs Melbourne in an accuracy/privacy conflict.